ads/zucks.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {validateData, writeScript} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function zucks(global, data) {
+  try {
+    validateData(data, ['frame_id']);
+    global.context.renderStart(data);
```

Comment:
It might be too early to call `renderStart` here.

`renderStart` means the ad response is back and the ad is ready to be displayed.
`noContentAvailable` means the ad response is back, but there's no ad to display.

I would expect Zucks SDK would somehow pass the info as events back here so that the script here can call the corresponding APIs at the right time.